tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -122,6 +129,12 @@ def get_all_fields(self):
         """Get all fields from the schema."""
         return self.fields
 
+    def get_field(self, name):
+        """Get a field from the schema by name."""
+        for field in self.fields:
+            if field.name == name:
+                return field
+
```

Comment:
I was prototyping this for fetching the startIndex given a name for a logging message. Could be useful in the future, might be worth deleting it as it is unused.